test_plugin/src/lib.rs

```diff
@@ -1,16 +1,18 @@
 extern crate deno_core;
 extern crate futures;
 
+use deno::plugin::register_op;
+use deno::plugin::InitContext;
```

Comment:
This seems to conflict with #5226. Do you plan to move some of these things to core?

Reply:
I think not core. Maybe it's own crate?

Reply:
> Maybe its own crate?

👍

Reply:
As per discussion with @ry, `mod plugin_api` has been moved to deno_core.
It may move to its own crate after the 1.0 release.